content/partials/types/_rest_client_options.textile

```diff
@@ -1,3 +1,3 @@
-- <span lang="default">queryTime</span><span lang="csharp,go">QueryTime</span><span lang="ruby">:query_time</span><span lang="python">query_time</span> := _false_ If true, the library will query the Ably servers for the current time when "issuing TokenRequests":/rest/authentication#token-authentication instead of relying on a locally-available time of day. Knowing the time accurately is needed to create valid signed Ably "TokenRequests":/rest/authentication#token-authentication, so this option is useful for library instances on auth servers where for some reason the server clock cannot be kept synchronized through normal means, such as an "NTP daemon":https://en.wikipedia.org/wiki/Ntpd . The server is queried for the current time once per client library instance (which stores the offset from the local clock), so if using this option you should avoid instancing a new version of the library for each request.<br>__Type: @Boolean@__
+- <span lang="default">queryTime</span><span lang="csharp,go">QueryTime</span><span lang="ruby">:query_time</span><span lang="python">query_time</span> := _false_ If true, the library will query the Ably servers for the current time when "issuing TokenRequests":/auth/token instead of relying on a locally-available time of day. Knowing the time accurately is needed to create valid signed Ably "TokenRequests":/api/rest-sdk/authentication#token-request, so this option is useful for library instances on auth servers where for some reason the server clock cannot be kept synchronized through normal means, such as an "NTP daemon":https://en.wikipedia.org/wiki/Ntpd . The server is queried for the current time once per client library instance (which stores the offset from the local clock), so if using this option you should avoid instancing a new version of the library for each request.<br>__Type: @Boolean@__
```

Comment:
Some links to token requests in push docs are going to /auth/token but here I see that you've redirected them to token-request specifically. No links are broken but just raising this as something I noticed.

Reply:
Good spot - I don't think this is an issue for now, but I've added it to the push ticket for when that section is reviewed.